registry/cdi/1.5.2/cdioperator.v1.5.2.clusterserviceversion.yaml

```diff
@@ -0,0 +1,127 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: cdi.v1.5.2
+  annotations:
+    tectonic-visibility: ocs
+spec:
+  displayName: CDI
+  description: |-
+     A declarative Kubernetes utility to import Virtual Machine images for use with Kubevirt
+
+  keywords: ['kubevirt', 'virtualization', 'virt', 'cdi']
+  maturity: alpha
+  version: 1.5.2
+  maintainers:
+  - name: RedHat
+    email: lveyde@redhat.com
+  installModes:
+  - type: OwnNamespace
+    supported: true
+  - type: SingleNamespace
+    supported: true
+  - type: MultiNamespace
+    supported: true
+  - type: AllNamespaces
+    supported: true
+  provider:
+    name: RedHat
+  labels:
+    operated-by: cdi
+  selector:
+    matchLabels:
+      operated-by: cdi
+  links:
+  - name: Learn more about the KubeVirt project
+    url: http://kubevirt.io/
+  - name: Documentation
+    url: https://github.com/kubevirt/containerized-data-importer/blob/master/README.md
+  - name: CDI Source Code
+    url: https://github.com/kubevirt/containerized-data-importer
+  - name: CDI Operator Source Code
+    url: https://github.com/kubevirt/cdi-operator
+
+  install:
+    strategy: deployment
+    spec:
+      permissions:
+      - serviceAccountName: cdi-operator
+        rules:
+        - apiGroups:
+          - cdi.kubevirt.io
+          resources:
+          - "*"
+          verbs:
+          - "*"
+        - apiGroups:
+          - ""
+          resources:
+          - pods
+          - services
+          - endpoints
+          - persistentvolumeclaims
+          - events
+          - configmaps
+          - secrets
+          verbs:
+          - "*"
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - "*"
+      deployments:
+      - name: cdi-operator
```

Comment:
This operator creates a deployment (when user creates a CR) that fails because it doesn't have enough permissions. We'll likely need to include that in either the CSV definition or as additional manifests.